Adding MorseCode and Projectile files including tests

emmyrunge opened the pull request.

Commit:
Adding MorseCode and Projectile files including tests

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -0,0 +1,57 @@
+    private final double radians;
+    private final double time;
+
+    public Projectile(double angle, double velocity, double time)
```
Time starts at 0. It should not be passed in.

andrewoid replied:
This method should not return a value.

And this code is not incrementing time.

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -0,0 +1,57 @@
+
+    }
+
+    public double incTime(double delta)
+    {
+
+        return time*delta;
+    }
```
This is wrong.

emmyrunge replied:
you said that time is incremented by delta, and that's what I'm doing here.

andrewoid replied:
This method should not return a value.

And this code is not incrementing time.

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -0,0 +1,57 @@
+
+    public double getY(double time)
+    {
+        return Math.sin(radians) * velocity * time - .5 * 9.8 * Math.pow(time, 2);
```
time * time

emmyrunge replied:
isn't the equation velocity * time? if that's not where you are referring to then where?

andrewoid replied:
Do time * time instead of using pow

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -0,0 +1,57 @@
+}
+
+
+/*
+for(int time = 0; time <= 10; time++)
+        {
+            System.out.println("x is: " + projectile.getX(time) + ", " + "y is: " + projectile.getY(time));
+        }
+ */
```
You don't need this commented out code.

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -0,0 +1,57 @@
+        return time;
+    }
+
+    public double getX(double time)
```
If time exists in the projectile now, why are you passing it in?

andrewoid commented on src/runge/projectile/MorseCode.java:
```diff
@@ -0,0 +1,44 @@
+public class MorseCode
+{
+    private final String[] morse;
+    String phrase;
```
What is "phrase"?

andrewoid commented on src/runge/projectile/MorseCodeTest.java:
```diff
@@ -0,0 +1,30 @@
+        String phrase = "is this gonna work?";
+
+        //then
+        morseCode.getPhrase(phrase);
```
You need to assertEquals() that the result of calling this method is what you expect it to be.

That's how you know if it works.

Commit:
Changed files according to comments

andrewoid commented on src/runge/projectile/Main.java:
```diff
@@ -0,0 +1,15 @@
+
+        System.out.println("x is: " + projectile.getX() + ", " + "y is: " + projectile.getY());
+
+        projectile.incTime(2);
```
Why are you increasing time here?

andrewoid commented on src/runge/projectile/Main.java:
```diff
@@ -0,0 +1,15 @@
+package runge.projectile;
+
+//import runge.projectile.Projectile;
```
Remove commented out code

andrewoid commented on src/runge/projectile/MorseCode.java:
```diff
@@ -0,0 +1,43 @@
+        this.english = english;
+    }
+
+    public StringBuilder getPhrase(String phrase)
```
Return a String. Not a StringBuilder

andrewoid replied:
The method was supposed to be called toMorseCode

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -0,0 +1,48 @@
+    private final double velocity;
+
+    private final double radians;
+    private double time = 6;
```
Why are you starting this off at 6?

andrewoid commented on src/runge/projectile/ProjectileTest.java:
```diff
@@ -0,0 +1,51 @@
+
+        //when
+        projectile.incTime(.0001);
+        int newTime = (int) projectile.getTime();
```
Why are you casting this?

andrewoid commented on src/runge/projectile/ProjectileTest.java:
```diff
@@ -0,0 +1,51 @@
+        int newTime = (int) projectile.getTime();
+
+        //then
+        assertEquals(6, newTime);
```
This obviously shouldn't be right.

Commit:
Corrected feedback for Projectile and MorseCode

andrewoid commented on src/runge/projectile/MorseCode.java:
```diff
@@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
             }
         }
-        return morseSentence;
+        return String.valueOf(morseSentence);
```
`morseSentence.toString()` instead

andrewoid commented on src/runge/projectile/MorseCode.java:
```diff
@@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
+                }
+            }
+        }
+        return String.valueOf(englishSentence);
```
`englishSentence.toString()`

andrewoid commented on src/runge/projectile/MorseCodeTest.java:
```diff
@@ -25,9 +25,33 @@ public void getPhrase()
 
         //when
-        String phrase = String.valueOf(morseCode.getPhrase("hey"));
+        String phrase = String.valueOf(morseCode.toMorseCode("hey"));
```
`String.valueOf()` is unnecessary

andrewoid commented on src/runge/projectile/Projectile.java:
```diff
@@ -37,12 +38,11 @@ public double getX()
     public void incTime(double delta)
     {
-        this.time += delta;
+        this.time = this.time + delta;
```
using `+=` was better

andrewoid commented on src/runge/projectile/MorseCode.java:
```diff
@@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
+
+        //break down sentence to word
+        //break down word to letter
+        for (int i = 0; i < morseCodeWord.length; i++)
```
What is `i` used for?

andrewoid commented on src/runge/projectile/MorseCodeTest.java:
```diff
@@ -25,9 +25,33 @@ public void getPhrase()
     }
+
+    @Test
+    public void toEnglish()
```
Try writing a test with multiple words.

emmyrunge commented:
I’ve been working on the last comment for a few hours now and can’t seem to
fix it. do you have time tonight to help?

On Tue, Feb 7, 2023 at 2:15 PM Andrew Schwimmer ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In src/runge/projectile/MorseCode.java
> <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099088976>:
>
> > @@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
>                  }
>              }
>          }
> -        return morseSentence;
> +        return String.valueOf(morseSentence);
>
> morseSentence.toString() instead
> ------------------------------
>
> In src/runge/projectile/MorseCode.java
> <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089219>:
>
> > +            else
> +            {
> +                for (int j = 0; j < morseCodeLetter.length; j++)
> +                {
> +                    for(int k = 0; k < morse.length; k++)
> +                    {
> +                        if (morseCodeLetter[j].equals(morse[k]))
> +                        {
> +                            englishSentence.append(english[k]);
> +                            break;
> +                        }
> +                    }
> +                }
> +            }
> +        }
> +        return String.valueOf(englishSentence);
>
> englishSentence.toString()
> ------------------------------
>
> In src/runge/projectile/MorseCodeTest.java
> <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089977>:
>
> > @@ -25,9 +25,33 @@ public void getPhrase()
>          MorseCode morseCode = new MorseCode(morse, english);
>
>          //when
> -        String phrase = String.valueOf(morseCode.getPhrase("hey"));
> +        String phrase = String.valueOf(morseCode.toMorseCode("hey"));
>
> String.valueOf() is unnecessary
> ------------------------------
>
> In src/runge/projectile/Projectile.java
> <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099090659>:
>
> >      }
>
>      public void incTime(double delta)
>      {
> -        this.time += delta;
> +        this.time = this.time + delta;
>
> using += was better
> ------------------------------
>
> In src/runge/projectile/MorseCode.java
> <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099091897>:
>
> > -        return morseSentence;
> +        return String.valueOf(morseSentence);
> +    }
> +
> +    public String toMessage(String morseCode)
> +    {
> +        StringBuilder englishSentence = new StringBuilder();
> +
> +        String [] morseCodeWord = morseCode.split("/");
> +
> +        String [] morseCodeLetter = morseCode.split(" , ");
> +
> +
> +        //break down sentence to word
> +        //break down word to letter
> +        for (int i = 0; i < morseCodeWord.length; i++)
>
> What is i used for?
> ------------------------------
>
> In src/runge/projectile/MorseCodeTest.java
> <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099092799>:
>
> >
>          //then
>          Assertions.assertEquals(" .... , . , -.-- , ", phrase);
>      }
> +
> +    @Test
> +    public void toEnglish()
>
> Try writing a test with multiple words.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emmyrunge/runge-mco152/pull/1#pullrequestreview-1287829864>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXCT23B2YVJUL3V52KCMTTDWWKNOTANCNFSM6AAAAAAUPM2WCE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

andrewoid commented:
Post your questions as a reply to my comment in PR instead of emailing me.

We should be able to meet tonight. What time?

On Tue, Feb 7, 2023, 5:41 PM emmy runge ***@***.***> wrote:

> I’ve been working on the last comment for a few hours now and can’t seem to
> fix it. do you have time tonight to help?
>
> On Tue, Feb 7, 2023 at 2:15 PM Andrew Schwimmer ***@***.***>
> wrote:
>
> > ***@***.**** commented on this pull request.
> > ------------------------------
> >
> > In src/runge/projectile/MorseCode.java
> > <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099088976
> >:
> >
> > > @@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
> > }
> > }
> > }
> > - return morseSentence;
> > + return String.valueOf(morseSentence);
> >
> > morseSentence.toString() instead
> > ------------------------------
> >
> > In src/runge/projectile/MorseCode.java
> > <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089219
> >:
> >
> > > + else
> > + {
> > + for (int j = 0; j < morseCodeLetter.length; j++)
> > + {
> > + for(int k = 0; k < morse.length; k++)
> > + {
> > + if (morseCodeLetter[j].equals(morse[k]))
> > + {
> > + englishSentence.append(english[k]);
> > + break;
> > + }
> > + }
> > + }
> > + }
> > + }
> > + return String.valueOf(englishSentence);
> >
> > englishSentence.toString()
> > ------------------------------
> >
> > In src/runge/projectile/MorseCodeTest.java
> > <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089977
> >:
> >
> > > @@ -25,9 +25,33 @@ public void getPhrase()
> > MorseCode morseCode = new MorseCode(morse, english);
> >
> > //when
> > - String phrase = String.valueOf(morseCode.getPhrase("hey"));
> > + String phrase = String.valueOf(morseCode.toMorseCode("hey"));
> >
> > String.valueOf() is unnecessary
> > ------------------------------
> >
> > In src/runge/projectile/Projectile.java
> > <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099090659
> >:
> >
> > > }
> >
> > public void incTime(double delta)
> > {
> > - this.time += delta;
> > + this.time = this.time + delta;
> >
> > using += was better
> > ------------------------------
> >
> > In src/runge/projectile/MorseCode.java
> > <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099091897
> >:
> >
> > > - return morseSentence;
> > + return String.valueOf(morseSentence);
> > + }
> > +
> > + public String toMessage(String morseCode)
> > + {
> > + StringBuilder englishSentence = new StringBuilder();
> > +
> > + String [] morseCodeWord = morseCode.split("/");
> > +
> > + String [] morseCodeLetter = morseCode.split(" , ");
> > +
> > +
> > + //break down sentence to word
> > + //break down word to letter
> > + for (int i = 0; i < morseCodeWord.length; i++)
> >
> > What is i used for?
> > ------------------------------
> >
> > In src/runge/projectile/MorseCodeTest.java
> > <https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099092799
> >:
> >
> > >
> > //then
> > Assertions.assertEquals(" .... , . , -.-- , ", phrase);
> > }
> > +
> > + @Test
> > + public void toEnglish()
> >
> > Try writing a test with multiple words.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/emmyrunge/runge-mco152/pull/1#pullrequestreview-1287829864
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXCT23B2YVJUL3V52KCMTTDWWKNOTANCNFSM6AAAAAAUPM2WCE
> >
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421570097>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA5VXQRBWGEM2E5G2TVCZTWWLFRFANCNFSM6AAAAAAUPM2WCE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

emmyrunge commented:
Will do. Does 9:30 work?

On Tue, Feb 7, 2023 at 5:55 PM Andrew Schwimmer ***@***.***>
wrote:

> Post your questions as a reply to my comment in PR instead of emailing me.
>
> We should be able to meet tonight. What time?
>
> On Tue, Feb 7, 2023, 5:41 PM emmy runge ***@***.***> wrote:
>
> > I’ve been working on the last comment for a few hours now and can’t seem
> to
> > fix it. do you have time tonight to help?
> >
> > On Tue, Feb 7, 2023 at 2:15 PM Andrew Schwimmer ***@***.***>
> > wrote:
> >
> > > ***@***.**** commented on this pull request.
> > > ------------------------------
> > >
> > > In src/runge/projectile/MorseCode.java
> > > <
> https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099088976
> > >:
> > >
> > > > @@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
> > > }
> > > }
> > > }
> > > - return morseSentence;
> > > + return String.valueOf(morseSentence);
> > >
> > > morseSentence.toString() instead
> > > ------------------------------
> > >
> > > In src/runge/projectile/MorseCode.java
> > > <
> https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089219
> > >:
> > >
> > > > + else
> > > + {
> > > + for (int j = 0; j < morseCodeLetter.length; j++)
> > > + {
> > > + for(int k = 0; k < morse.length; k++)
> > > + {
> > > + if (morseCodeLetter[j].equals(morse[k]))
> > > + {
> > > + englishSentence.append(english[k]);
> > > + break;
> > > + }
> > > + }
> > > + }
> > > + }
> > > + }
> > > + return String.valueOf(englishSentence);
> > >
> > > englishSentence.toString()
> > > ------------------------------
> > >
> > > In src/runge/projectile/MorseCodeTest.java
> > > <
> https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089977
> > >:
> > >
> > > > @@ -25,9 +25,33 @@ public void getPhrase()
> > > MorseCode morseCode = new MorseCode(morse, english);
> > >
> > > //when
> > > - String phrase = String.valueOf(morseCode.getPhrase("hey"));
> > > + String phrase = String.valueOf(morseCode.toMorseCode("hey"));
> > >
> > > String.valueOf() is unnecessary
> > > ------------------------------
> > >
> > > In src/runge/projectile/Projectile.java
> > > <
> https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099090659
> > >:
> > >
> > > > }
> > >
> > > public void incTime(double delta)
> > > {
> > > - this.time += delta;
> > > + this.time = this.time + delta;
> > >
> > > using += was better
> > > ------------------------------
> > >
> > > In src/runge/projectile/MorseCode.java
> > > <
> https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099091897
> > >:
> > >
> > > > - return morseSentence;
> > > + return String.valueOf(morseSentence);
> > > + }
> > > +
> > > + public String toMessage(String morseCode)
> > > + {
> > > + StringBuilder englishSentence = new StringBuilder();
> > > +
> > > + String [] morseCodeWord = morseCode.split("/");
> > > +
> > > + String [] morseCodeLetter = morseCode.split(" , ");
> > > +
> > > +
> > > + //break down sentence to word
> > > + //break down word to letter
> > > + for (int i = 0; i < morseCodeWord.length; i++)
> > >
> > > What is i used for?
> > > ------------------------------
> > >
> > > In src/runge/projectile/MorseCodeTest.java
> > > <
> https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099092799
> > >:
> > >
> > > >
> > > //then
> > > Assertions.assertEquals(" .... , . , -.-- , ", phrase);
> > > }
> > > +
> > > + @Test
> > > + public void toEnglish()
> > >
> > > Try writing a test with multiple words.
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/emmyrunge/runge-mco152/pull/1#pullrequestreview-1287829864
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AXCT23B2YVJUL3V52KCMTTDWWKNOTANCNFSM6AAAAAAUPM2WCE
> > >
> > > .
> > > You are receiving this because you authored the thread.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421570097>,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAA5VXQRBWGEM2E5G2TVCZTWWLFRFANCNFSM6AAAAAAUPM2WCE
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421582323>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXCT23CZA4Z3Z3KDTFOD6N3WWLHHRANCNFSM6AAAAAAUPM2WCE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

andrewoid commented:
Yes.

On Tue, Feb 7, 2023, 6:13 PM emmy runge ***@***.***> wrote:

> Will do. Does 9:30 work?
>
> On Tue, Feb 7, 2023 at 5:55 PM Andrew Schwimmer ***@***.***>
> wrote:
>
> > Post your questions as a reply to my comment in PR instead of emailing
> me.
> >
> > We should be able to meet tonight. What time?
> >
> > On Tue, Feb 7, 2023, 5:41 PM emmy runge ***@***.***> wrote:
> >
> > > I’ve been working on the last comment for a few hours now and can’t
> seem
> > to
> > > fix it. do you have time tonight to help?
> > >
> > > On Tue, Feb 7, 2023 at 2:15 PM Andrew Schwimmer ***@***.***>
> > > wrote:
> > >
> > > > ***@***.**** commented on this pull request.
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCode.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099088976
> > > >:
> > > >
> > > > > @@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
> > > > }
> > > > }
> > > > }
> > > > - return morseSentence;
> > > > + return String.valueOf(morseSentence);
> > > >
> > > > morseSentence.toString() instead
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCode.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089219
> > > >:
> > > >
> > > > > + else
> > > > + {
> > > > + for (int j = 0; j < morseCodeLetter.length; j++)
> > > > + {
> > > > + for(int k = 0; k < morse.length; k++)
> > > > + {
> > > > + if (morseCodeLetter[j].equals(morse[k]))
> > > > + {
> > > > + englishSentence.append(english[k]);
> > > > + break;
> > > > + }
> > > > + }
> > > > + }
> > > > + }
> > > > + }
> > > > + return String.valueOf(englishSentence);
> > > >
> > > > englishSentence.toString()
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCodeTest.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089977
> > > >:
> > > >
> > > > > @@ -25,9 +25,33 @@ public void getPhrase()
> > > > MorseCode morseCode = new MorseCode(morse, english);
> > > >
> > > > //when
> > > > - String phrase = String.valueOf(morseCode.getPhrase("hey"));
> > > > + String phrase = String.valueOf(morseCode.toMorseCode("hey"));
> > > >
> > > > String.valueOf() is unnecessary
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/Projectile.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099090659
> > > >:
> > > >
> > > > > }
> > > >
> > > > public void incTime(double delta)
> > > > {
> > > > - this.time += delta;
> > > > + this.time = this.time + delta;
> > > >
> > > > using += was better
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCode.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099091897
> > > >:
> > > >
> > > > > - return morseSentence;
> > > > + return String.valueOf(morseSentence);
> > > > + }
> > > > +
> > > > + public String toMessage(String morseCode)
> > > > + {
> > > > + StringBuilder englishSentence = new StringBuilder();
> > > > +
> > > > + String [] morseCodeWord = morseCode.split("/");
> > > > +
> > > > + String [] morseCodeLetter = morseCode.split(" , ");
> > > > +
> > > > +
> > > > + //break down sentence to word
> > > > + //break down word to letter
> > > > + for (int i = 0; i < morseCodeWord.length; i++)
> > > >
> > > > What is i used for?
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCodeTest.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099092799
> > > >:
> > > >
> > > > >
> > > > //then
> > > > Assertions.assertEquals(" .... , . , -.-- , ", phrase);
> > > > }
> > > > +
> > > > + @Test
> > > > + public void toEnglish()
> > > >
> > > > Try writing a test with multiple words.
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/emmyrunge/runge-mco152/pull/1#pullrequestreview-1287829864
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AXCT23B2YVJUL3V52KCMTTDWWKNOTANCNFSM6AAAAAAUPM2WCE
> > > >
> > > > .
> > > > You are receiving this because you authored the thread.Message ID:
> > > > ***@***.***>
> > > >
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421570097
> >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AAA5VXQRBWGEM2E5G2TVCZTWWLFRFANCNFSM6AAAAAAUPM2WCE
> > >
> > > .
> > > You are receiving this because you commented.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421582323>,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXCT23CZA4Z3Z3KDTFOD6N3WWLHHRANCNFSM6AAAAAAUPM2WCE
> >
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421601907>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA5VXUHXSM3DHUGJ7T4RCTWWLJJFANCNFSM6AAAAAAUPM2WCE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

andrewoid commented:
https://zoom.us/s/4616736274

On Tue, Feb 7, 2023 at 6:13 PM emmy runge ***@***.***> wrote:

> Will do. Does 9:30 work?
>
> On Tue, Feb 7, 2023 at 5:55 PM Andrew Schwimmer ***@***.***>
> wrote:
>
> > Post your questions as a reply to my comment in PR instead of emailing
> me.
> >
> > We should be able to meet tonight. What time?
> >
> > On Tue, Feb 7, 2023, 5:41 PM emmy runge ***@***.***> wrote:
> >
> > > I’ve been working on the last comment for a few hours now and can’t
> seem
> > to
> > > fix it. do you have time tonight to help?
> > >
> > > On Tue, Feb 7, 2023 at 2:15 PM Andrew Schwimmer ***@***.***>
> > > wrote:
> > >
> > > > ***@***.**** commented on this pull request.
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCode.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099088976
> > > >:
> > > >
> > > > > @@ -37,7 +35,42 @@ public StringBuilder getPhrase(String phrase)
> > > > }
> > > > }
> > > > }
> > > > - return morseSentence;
> > > > + return String.valueOf(morseSentence);
> > > >
> > > > morseSentence.toString() instead
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCode.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089219
> > > >:
> > > >
> > > > > + else
> > > > + {
> > > > + for (int j = 0; j < morseCodeLetter.length; j++)
> > > > + {
> > > > + for(int k = 0; k < morse.length; k++)
> > > > + {
> > > > + if (morseCodeLetter[j].equals(morse[k]))
> > > > + {
> > > > + englishSentence.append(english[k]);
> > > > + break;
> > > > + }
> > > > + }
> > > > + }
> > > > + }
> > > > + }
> > > > + return String.valueOf(englishSentence);
> > > >
> > > > englishSentence.toString()
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCodeTest.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099089977
> > > >:
> > > >
> > > > > @@ -25,9 +25,33 @@ public void getPhrase()
> > > > MorseCode morseCode = new MorseCode(morse, english);
> > > >
> > > > //when
> > > > - String phrase = String.valueOf(morseCode.getPhrase("hey"));
> > > > + String phrase = String.valueOf(morseCode.toMorseCode("hey"));
> > > >
> > > > String.valueOf() is unnecessary
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/Projectile.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099090659
> > > >:
> > > >
> > > > > }
> > > >
> > > > public void incTime(double delta)
> > > > {
> > > > - this.time += delta;
> > > > + this.time = this.time + delta;
> > > >
> > > > using += was better
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCode.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099091897
> > > >:
> > > >
> > > > > - return morseSentence;
> > > > + return String.valueOf(morseSentence);
> > > > + }
> > > > +
> > > > + public String toMessage(String morseCode)
> > > > + {
> > > > + StringBuilder englishSentence = new StringBuilder();
> > > > +
> > > > + String [] morseCodeWord = morseCode.split("/");
> > > > +
> > > > + String [] morseCodeLetter = morseCode.split(" , ");
> > > > +
> > > > +
> > > > + //break down sentence to word
> > > > + //break down word to letter
> > > > + for (int i = 0; i < morseCodeWord.length; i++)
> > > >
> > > > What is i used for?
> > > > ------------------------------
> > > >
> > > > In src/runge/projectile/MorseCodeTest.java
> > > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#discussion_r1099092799
> > > >:
> > > >
> > > > >
> > > > //then
> > > > Assertions.assertEquals(" .... , . , -.-- , ", phrase);
> > > > }
> > > > +
> > > > + @Test
> > > > + public void toEnglish()
> > > >
> > > > Try writing a test with multiple words.
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/emmyrunge/runge-mco152/pull/1#pullrequestreview-1287829864
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AXCT23B2YVJUL3V52KCMTTDWWKNOTANCNFSM6AAAAAAUPM2WCE
> > > >
> > > > .
> > > > You are receiving this because you authored the thread.Message ID:
> > > > ***@***.***>
> > > >
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> > https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421570097
> >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AAA5VXQRBWGEM2E5G2TVCZTWWLFRFANCNFSM6AAAAAAUPM2WCE
> > >
> > > .
> > > You are receiving this because you commented.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421582323>,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXCT23CZA4Z3Z3KDTFOD6N3WWLHHRANCNFSM6AAAAAAUPM2WCE
> >
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emmyrunge/runge-mco152/pull/1#issuecomment-1421601907>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA5VXUHXSM3DHUGJ7T4RCTWWLJJFANCNFSM6AAAAAAUPM2WCE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

Commit:
Corrected comments primarily focused on MorseCode
